src/main/kotlin/com/hitc/hitweconomyplugin/main/utils/GameFileUtils.kt

```diff
@@ -1,40 +1,26 @@
 package com.hitc.hitweconomyplugin.main.utils
 
-import com.hitc.hitweconomyplugin.main.core.DataFile
-import com.hitc.hitweconomyplugin.main.core.EPlayer
-import com.hitc.hitweconomyplugin.main.core.PlayerData
-import com.hitc.hitweconomyplugin.main.core.Score
+import com.hitc.hitweconomyplugin.main.core.*
 import kotlinx.serialization.decodeFromString
-import kotlinx.serialization.encodeToString
 import kotlinx.serialization.json.Json
 import kotlinx.serialization.json.decodeFromJsonElement
-import kotlinx.serialization.json.encodeToJsonElement
 import org.bukkit.entity.Player
 import java.io.File
 import java.nio.file.Files
 import java.util.*
-import java.util.zip.DataFormatException
 import kotlin.Exception
 
 object GameFileUtils {
 
-    fun loadScores(ePlayer: EPlayer,
-                   calendar: Calendar = Calendar.getInstance(TimeZone.getTimeZone("EST"))
-    ) : MutableList<Score?> {
-        var scores = MutableList<Score?>(0) { null }
+    fun loadScores(f: File) : Scores {
+        var scores = Scores(emptyList<Score>().toMutableList(), f)
```

Comment:
I knew there was a better way to do that
You can either do `arrayListOf()` or `ArrayList()` instead of that `emptyList<Score>().toMutableList()`

Reply:
even `mutableListOf()` works